packages/core/src/node/console-logger-server.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
```

Comment:
`2018 TypeFox`?

Reply:
No. This module already exists on the `master`.

https://github.com/theia-ide/theia/blob/4f42f16930af549e295a0390a23944c195d58b84/packages/core/src/common/console-logger-server.ts#L1-L6